test/unit/org/apache/cassandra/db/DirectoriesTest.java

```diff
@@ -23,18 +23,30 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.*;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.Callable;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 
 import com.google.common.collect.Sets;
 import org.apache.commons.lang3.StringUtils;
 
+import org.junit.After;
 import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+// Our version of Sfl4j seems to be missing the ListAppender class.
```

Comment:
I think these comments can be removed.

Reply:
@henrikingo and I discussed this, and agreed there's value in the comment, bc we don't usually directly use the logging impl classes.